.github/workflows/ci.yml

```
@@ -5,34 +5,40 @@ on: [ push, pull_request ]
 jobs:
    build:
       runs-on: ubuntu-latest
+      outputs:
+         nvmrc: ${{ steps.makeNodeVersionOutput.outputs.nvmrc }}
       steps:
-         -
-            uses: actions/checkout@v3
+         -  uses: actions/checkout@v4
             with:
                fetch-depth: 0 # Fetch all history
-         -
-            uses: actions/setup-node@v3
+         -  uses: actions/setup-node@v4
             with:
                node-version-file: '.nvmrc'
-         - run: npm i -g npm@8.5.5
-         - run: npm ci
-         - run: npm run standards
+         -  name: Put NVM version in output
+            id: makeNodeVersionOutput
+            run: echo "nvmrc=$(cat .nvmrc)" >> "$GITHUB_OUTPUT"
+         -  run: npm ci
```

For single keys, we'll typically only have one space after the dash.

```suggestion
         - run: npm ci
```